Curtains/Curtains.Domain/Data/Mappings/ReviewMapping.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Curtains.Domain.Models
```

Comment:
We still need change mapping's namespaces from Curtains.Domain.Models to Curtains.Domain.Data

Reply:
Not just ReveiwMapping but all of them)